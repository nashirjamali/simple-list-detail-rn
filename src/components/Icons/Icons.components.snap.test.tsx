import React from 'react';
import {render} from '@testing-library/react-native';

import ArrowDownIcon from './ArrowDownIcon.icons';
import CopyIcon from './CopyIcon.icons';
import DotIcon from './DotIcon.icons';
import ArrowRightIcon from './ArrowRightIcon.icons';

describe('Icon Components', () => {
  it('renders ArrowDownIcon correctly', () => {
    const {toJSON} = render(<ArrowDownIcon color="black" />);
    expect(toJSON()).toMatchSnapshot();
  });

  it('renders CopyIcon correctly', () => {
    const {toJSON} = render(<CopyIcon color="black" />);
    expect(toJSON()).toMatchSnapshot();
  });

  it('renders DotIcon correctly', () => {
    const {toJSON} = render(<DotIcon color="black" />);
    expect(toJSON()).toMatchSnapshot();
  });

  it('renders ArrowRightIcon correctly', () => {
    const {toJSON} = render(<ArrowRightIcon color="black" />);
    expect(toJSON()).toMatchSnapshot();
  });
});
