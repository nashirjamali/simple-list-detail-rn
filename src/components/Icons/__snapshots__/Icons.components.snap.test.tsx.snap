// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon Components renders ArrowDownIcon correctly 1`] = `
<RNSVGSvgView
  align="xMidYMid"
  bbHeight="100%"
  bbWidth="100%"
  color="black"
  focusable={false}
  meetOrSlice={0}
  minX={0}
  minY={0}
  style={
    [
      {
        "backgroundColor": "transparent",
        "borderWidth": 0,
      },
      {
        "flex": 0,
        "height": "100%",
        "width": "100%",
      },
    ]
  }
  vbHeight={48}
  vbWidth={48}
>
  <RNSVGGroup
    fill={
      {
        "payload": 4278190080,
        "type": 0,
      }
    }
  >
    <RNSVGPath
      d="M0 0h48v48H0z"
      fill={null}
      propList={
        [
          "fill",
        ]
      }
    />
    <RNSVGPath
      d="M24 29.171L9.414 14.585 6.586 17.413 24 34.827 41.414 17.413 38.586 14.585z"
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "fill",
        ]
      }
    />
  </RNSVGGroup>
</RNSVGSvgView>
`;

exports[`Icon Components renders ArrowRightIcon correctly 1`] = `
<RNSVGSvgView
  align="xMidYMid"
  bbHeight={20}
  bbWidth={20}
  color="black"
  fill="none"
  focusable={false}
  height={20}
  meetOrSlice={0}
  minX={0}
  minY={0}
  style={
    [
      {
        "backgroundColor": "transparent",
        "borderWidth": 0,
      },
      {
        "flex": 0,
        "height": 20,
        "width": 20,
      },
    ]
  }
  vbHeight={24}
  vbWidth={24}
  width={20}
>
  <RNSVGGroup
    fill={null}
    propList={
      [
        "fill",
      ]
    }
  >
    <RNSVGPath
      d="M13.233 16.457a.75.75 0 101.034 1.086l-1.034-1.086zm6.284-3.914a.75.75 0 10-1.034-1.086l1.034 1.086zm-1.034 0a.75.75 0 101.034-1.086l-1.034 1.086zm-4.216-6.086a.75.75 0 10-1.034 1.086l1.034-1.086zM19 12.75a.75.75 0 000-1.5v1.5zm-14-1.5a.75.75 0 000 1.5v-1.5zm9.267 6.293l5.25-5-1.034-1.086-5.25 5 1.034 1.086zm5.25-6.086l-5.25-5-1.034 1.086 5.25 5 1.034-1.086zM19 11.25H5v1.5h14v-1.5z"
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "fill",
        ]
      }
    />
  </RNSVGGroup>
</RNSVGSvgView>
`;

exports[`Icon Components renders CopyIcon correctly 1`] = `
<RNSVGSvgView
  align="xMidYMid"
  bbHeight="800px"
  bbWidth="800px"
  color="black"
  fill="none"
  focusable={false}
  height="800px"
  meetOrSlice={0}
  minX={0}
  minY={0}
  style={
    [
      {
        "backgroundColor": "transparent",
        "borderWidth": 0,
      },
      {
        "flex": 0,
        "height": 800,
        "width": 800,
      },
    ]
  }
  vbHeight={24}
  vbWidth={24}
  width="800px"
>
  <RNSVGGroup
    fill={null}
    propList={
      [
        "fill",
      ]
    }
  >
    <RNSVGPath
      d="M6 11c0-2.828 0-4.243.879-5.121C7.757 5 9.172 5 12 5h3c2.828 0 4.243 0 5.121.879C21 6.757 21 8.172 21 11v5c0 2.828 0 4.243-.879 5.121C19.243 22 17.828 22 15 22h-3c-2.828 0-4.243 0-5.121-.879C6 20.243 6 18.828 6 16v-5z"
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "stroke",
          "strokeWidth",
        ]
      }
      stroke={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      strokeWidth={1.5}
    />
    <RNSVGPath
      d="M6 19a3 3 0 01-3-3v-6c0-3.771 0-5.657 1.172-6.828C5.343 2 7.229 2 11 2h4a3 3 0 013 3"
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "stroke",
          "strokeWidth",
        ]
      }
      stroke={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      strokeWidth={1.5}
    />
  </RNSVGGroup>
</RNSVGSvgView>
`;

exports[`Icon Components renders DotIcon correctly 1`] = `
<RNSVGSvgView
  align="xMidYMid"
  bbHeight={20}
  bbWidth={20}
  color="black"
  fill="none"
  focusable={false}
  height={20}
  meetOrSlice={0}
  minX={0}
  minY={0}
  style={
    [
      {
        "backgroundColor": "transparent",
        "borderWidth": 0,
      },
      {
        "flex": 0,
        "height": 20,
        "width": 20,
      },
    ]
  }
  vbHeight={24}
  vbWidth={24}
  width={20}
>
  <RNSVGGroup
    fill={null}
    propList={
      [
        "fill",
      ]
    }
  >
    <RNSVGPath
      d="M12 9.5a2.5 2.5 0 110 5 2.5 2.5 0 010-5z"
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "fill",
        ]
      }
    />
  </RNSVGGroup>
</RNSVGSvgView>
`;
